### PR TITLE
Fix #41 unable to integrate test target

### PR DESCRIPTION
**Pull request by joshuatbrown**
Here's what we need to figure out:

1. Should `RZBluetoothMock` be its own framework/pod, or should we continue using subspecs for Core and Mock since the real problem is with Test?

2. How has 1.2.2 been released but not merged to `master`?

3. How do we get the tests to build and pass? This all works as a pod dependency in our project, but now the RZBluetooth tests don't build.

### CODE REVIEW

**Comment by mplorentz** [comment]
1. I think `RZBluetooth/Mock` should stay a subspec. No need to break things there. In fact, maybe `RZBluetooth/Test` should remain a subspec as well. It's confusing but so is the reason that we are making a separate Podspec. Probably a question for Brian King.

2. It looks like 1.2.2 is just one commit away from `master`, where the version number was incremented. Weird, but shouldn't cause issues.

3. See #84 and #86. Maybe we should go ahead and fix that issue ourselves as well?

**Review by mplorentz — COMMENTED** [review]
No other comments besides the one I made above.

File: RZBluetoothTests/RZBSimulatedTestCase.h [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
+#import "RZBluetooth/RZBMockPeripheralManager.h"
 
-#import "RZBSimulatedDevice.h"
-#import "RZBSimulatedCentral.h"
-#import "RZBSimulatedConnection.h"
-#import "RZBSimulatedCallback.h"
+#import "RZBluetooth/RZBSimulatedDevice.h"
+#import "RZBluetooth/RZBSimulatedCentral.h"
+#import "RZBluetooth/RZBSimulatedConnection.h"
+#import "RZBluetooth/RZBSimulatedCallback.h"
```
Comment (cpatterson-lilly) [inline RZBluetoothTests/RZBSimulatedTestCase.h:22]: I would have expected the headers in lines 15-23 to be in the RZBluetoothMock framework?

File: RZBluetoothTests/RZBSimulatedTestCase.m [hunk 2]
```diff
@@ -7,11 +7,11 @@
 //
 
 #import "RZBTestDefines.h"
-#import "RZMockBluetooth.h"
+#import "RZBluetooth/RZMockBluetooth.h"
```
Comment (cpatterson-lilly) [inline RZBluetoothTests/RZBSimulatedTestCase.m:10]: Ditto for line 10 here.

**Comment by joshuatbrown** [comment]
I'm going to close this PR and reopen it in `Raizlabs/RZBluetooth`.

**Review by MarcSteven — COMMENTED** [review]
Cool

**Pull request closed without merging**